Backend/routes/eldenRingApi.ts

```diff
@@ -1,15 +1,15 @@
-import axios, { AxiosInstance } from 'axios';
+import { gql, GraphQLClient } from 'graphql-request';
 import logging from '../utils/logging';
-import { gql, GraphQLClient, request, Variables } from 'graphql-request';
 
-const NAMESPACE: string = "ERAPI";
+const NAMESPACE = 'ERAPI';
 
 export default class ERApiController {
+  private client: GraphQLClient = new GraphQLClient('https://eldenring.fanapis.com/api/graphql');
 
-    private client: GraphQLClient = new GraphQLClient('https://eldenring.fanapis.com/api/graphql');
+  private graphQL = gql;
 
-    private searchQuery(route: string, name: string) : string {
-        return gql`
+  private searchQuery(route: string, name: string): string {
+    return this.graphQL`
```

Comment:
Not sure why this was changed but the other queries were left alone

Reply:
Bizarre - must've been an auto-lint, was switching around the rules quite often as I was making commits. Can be changed back!

Reply:
Possibly, not a big deal but it did also create a new private member just to do this which I haven't seen a linter do before lol